languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/hunspell/spelling.txt

```diff
@@ -996,4 +994,46 @@ rooftop
 plug-and-play
 plug and play
 scraping
-Colisée
\ No newline at end of file
+Colisée
+bazin
+yodleur
+yodleurs
+yodleuse
+yodleuses
+Yverdon
+Paul Signac
+IST
```

Comment:
reminder: please add abbreviations and uncommon names to the `ignore.txt` (there are a couple more below)

Reply:
@tiff They are here on purpose, people use these as words quite commonly :)

Reply:
OK Got it